musicbrainzngs/mbxml.py

```diff
@@ -523,6 +525,9 @@ def parse_cdstub(cdstub):
 
     return result
 
+def parse_offset_list(ol):
```

Comment:
Can we have an updated test (or a test for this method if it doesn't already exist)